openshift/forms-flow-ai-charts/charts/forms-flow-ai/values.yaml

```diff
@@ -13,82 +20,189 @@ forms-flow-forms:
 forms-flow-idm:
   keycloak:
     ingress:
-      hostname: epd-keycloak-test.apps.silver.devops.gov.bc.ca
+      hostname: forms-flow-idm-{{.Release.Namespace}}.{{tpl (.Values.Domain) .}}
   realm: forms-flow-ai
 
 forms-flow-web:
-  clientid: user-service
+  clientid: forms-flow-web
+
+forms-flow-auth:
+  imagesecret: "ewogICJhdXRocyI6IHsKICAgICJkb2NrZXIuaW8iOiB7CiAgICAgICJ1c2VybmFtZSI6ICJ0ZXN0YWNjb3VudCIsCiAgICAgICJwYXNzd29yZCI6ICJ0ZXN0X2FjY291bnRfc2FtcGxlX2RvY2tlcmFjY291bnQiLAogICAgICAiYXV0aCI6ICJhR1ZzYkc4Z2QyOXliR1E9IgogICAgfQogIH0sCiAgImFsZyI6ICJIUzI1NiIKfQ=="
+
 
 mongodb:
-  arbiter: 
-    enabled: false
-    containerSecurityContext:
-      enabled: false
-    podSecurityContext:
-      enabled: false
+  image:
+    tag: 4.2.21-debian-10-r8
+  enabled: true
+  fullnameOverride: "forms-flow-ai-mongodb"
+  clusterDomain: cluster.local
+  architecture: replicaset
+  useStatefulSet: true
   auth:
     databases: 
       - formsflow 
     passwords:
       - mongodb
     usernames:
       - mongodb
+    enabled: true
+    rootUser: root
+    rootPassword: "aot123"
```

Comment:
Secrets here. 

At the simplest we could put something in like an invalid character that MUST be changed before executing.  eg  instead of a string, we could store `-MUST CHANGE-` which would be invalid yaml and fail, but at least would prompt the user to change before executing.